src/Cache/CollectionCacheKey.php

```diff
@@ -43,14 +43,14 @@ class CollectionCacheKey extends CacheKey
      * @param string               $association     The field name that represents the association.
      * @param array<string, mixed> $ownerIdentifier The identifier of the owning entity.
      */
-    public function __construct($entityClass, $association, array $ownerIdentifier)
+    public function __construct($entityClass, $association, array $ownerIdentifier, string $filterHash)
```

Comment:
Let's avoid this breaking change.

```suggestion
    public function __construct($entityClass, $association, array $ownerIdentifier, string $filterHash = '')
```